test/test_integration.py

```diff
@@ -217,6 +217,34 @@ def test_upload_gallery(self):
 
         self.snapshot('test_upload_gallery')
 
+    def test_vertical_slider(self):
+        app = dash.Dash(__name__)
+
+        app.layout = html.Div([
+            html.Label('Vertical Slider'),
+            dcc.Slider(
+                id='vertical-slider',
+                min=0,
+                max=9,
+                marks={i: 'Label {}'.format(i) if i == 1 else str(i)
+                       for i in range(1, 6)},
+                value=5,
+                vertical=True,
+            ),
+        ], style={'height': '500px'})
+        self.startServer(app)
+
+        self.wait_for_element_by_css_selector('#vertical-slider')
+        self.snapshot('vertical slider')
+
+        v_slider = self.driver.find_element_by_css_selector(
+            '#vertical-slider div[role="slider"]'
+        )
+        v_slider.click()
+
+        for entry in self.log_facade.get_log():
+            raise Exception('browser error logged during test', entry)
```

Comment:
If there's a message here, something went wrong. Not doing this for all tests as lots of tests have issues at the moment..

Reply:
https://github.com/plotly/dash-core/issues/51

Reply:
I'm wondering if this test shouldn't be in the `test_gallery` test, since that's where most of the snapshots of components are taken. Although, I would rather see the `test_gallery` test eventually be broken up into specific tests per component, so I guess this is a good start! :-)

Reply:
I think the gallery snapshot approach is a bad idea in general. When I see a diff I want that diff to be isolated and linked to as little dependencies as possible. If the vertical slider breaks, it's obvious. And no wondering if there's multiple issues in the single diff.

Reply:
Agreed - so many times I've had to look at a diff where something was added in the middle of a long page... and percy has no idea that it's all the same content just shifted down, so I have to try and do that diffing in my head. Much better to break these up into single-purpose images, and not worry about making a huge total number of images.